app/src/main/res/layout/fragment_edit_note.xml

```diff
@@ -11,22 +11,28 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:theme="@style/Widget.Boardbuddies.Toolbar"
-        app:layout_constraintTop_toTopOf="parent"
+        app:contentInsetStartWithNavigation="0dp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
         app:navigationIcon="@drawable/baseline_arrow_back_24"
-        app:contentInsetStartWithNavigation="0dp"
         app:titleMarginStart="0dp">
 
-        <TextView
-            android:id="@+id/note_title"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="placeholder"
-            android:textColor="#F5F5F5"
-            android:textSize="22sp"
-            android:textStyle="bold" />
-
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+            <ImageButton
+                android:id="@+id/delete_icon"
+                android:layout_width="0dp"
+                android:layout_height="fill_parent"
```

Comment:
1. fill_parent is deprecated in favor of match_parent
2. match_parent doesn't work within a constraint layout, you should use 0dp instead so it can take as much space as it needs (see https://stackoverflow.com/questions/37603751/set-width-to-match-constraints-in-constraintlayout)